cpu/esp32/include/irq_arch.h

```diff
@@ -50,6 +50,7 @@ extern "C" {
 #define CPU_INUM_SYSTIMER   20  /**< Level interrupt with medium priority 2 */
 #define CPU_INUM_BLE        21  /**< Level interrupt with medium priority 2 */
 #define CPU_INUM_CACHEERR   25  /**< Level interrupt with high priority 4 */
+#define CPU_INUM_SERIAL_JTAG 26 /**< Level interrupt with low priority 1 */
```

Comment:
The level specifies the priority of the the interrupts which is flexible for RISC-V based ESP32x SoCs but fixed for Xtensa-based SoCs. The value given here is the fixed value as used by the Xtensa-based SoCs which is 5.
```suggestion
#define CPU_INUM_SERIAL_JTAG 26 /**< Level interrupt with high priority 5 */
```

Reply:
We should check whether we could use another interrupt because level 5 is the highest one and higher than the cache error or WDT interrupts which have a level of 4.

If I'm not wrong, level 1 interrupt 10 should be free.